disnake/ui/item.py

```diff
@@ -42,7 +42,7 @@ class WrappedComponent(ABC):
     The following classes implement this ABC:
 
     - :class:`disnake.ui.Button`
-    - :class:`disnake.ui.Select`
+    - subtypes of :class:`disnake.ui.BaseSelect` (:class:`~disnake.ui.Select`)
```

Comment:
this should display `ui.Select` *at minimum* and perhaps the full name.

Reply:
https://github.com/DisnakeDev/disnake/pull/800/commits/a6694525818becff893c2fb0909435f6f10fe863